package Task3;

import java.io.IOException;

public class main3 {
    public static void main (String[] args) throws IOException {
        Person p1= new Person();
        Person p2= new Person();
        Person p4= new Person();
        Person p3= new Person();
        Person p5= new Person();
        p1.input();
        p2.input();
        p3.input();
        p4.input();
        p5.input();
        p1.age();
        p2.age();
        p3.age();
        p4.age();
        p5.age();
        p1.output();
        p2.output();
        p3.output();
        p4.output();
        p5.output();
        p1.changeName();
        p1.output();

    }
}
